sdcm/sct_events/base.py

```diff
@@ -231,6 +268,10 @@ def dont_publish(self):
     def ready_to_publish(self):
         self._ready_to_publish = True
 
+    @cached_property
+    def event_to_json(self):
```

Comment:
the name of this is very confusing, i.e. it's not clear how it's different from `to_json`,

maybe have a `subcontext_field` that you could use to filter the output of `to_json` of an event.

Reply:
changed